com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/model/ParameterMetadata.java

```diff
@@ -11,23 +11,17 @@
 @JsonDeserialize(as = ImmutableParameterMetadata.class)
 public interface ParameterMetadata {
 
-    @Nullable
```

Comment:
They are required now?

Reply:
Yes, it wouldn't make sense to have a parameter without an ID, so it should not be allowed at all.